### DISCUSSION TITLE
【灌水】睡不着了，小溜一会绳网

### DISCUSSION THREAD

**Discussion by hzjwork** [share121/inter-knot#1177] [其他]
突然发现加载速度变快了好多，而且还能无限往下滑动了

**Comment by sfhjhsdfg** [comment]
听说之前绳网维护导致信息删除了，也不知道发生了什么事，听周围的绳匠朋友说好像遭黑客入侵了，现在的绳网简直和新建的一样，希望早些恢复往日荣光吧
